_docker/drupal/README.md

```diff
@@ -106,6 +106,28 @@ section of composer.json:
 }
 ```
 
+### Updating Metadata with the Metatag module for Articles and Topics
+
+Metatag is used to automatically generate structured metadata. To manipulate generated metadata, you will need to work with the Metatag module and export the settings.
+
+1. Locally spin up an instance of drupal (i.e. bundle exec ./control.rb -u --run-the-stack)
+2. Login to drupal with the admin username and password.
+3. Navigate to Configuration>Search and metadata>Metatag 
+4. Select the 'Edit' button next to either 'Content: Article' or 'Content: Topic' and manipulate data to fit requirements. (Browse all available tokens can be helpful when working with dynamic variables)
+5. Save all changes.
+
+Now that all changes have been saved, it is necessary to export the configuration in order for subsequent builds to contain the changes.
```

Comment:
This is not needed, though having something about having to edit the global defaults would be good.